src/test/resources/application.conf

```diff
@@ -0,0 +1,8 @@
+akka {
+  loggers = ["de.heikoseeberger.akkalog4j.Log4jLogger"]
+  loglevel = "DEBUG"
+  test {
+    timefactor = 10
+    timefactor = ${?TEST_TIME_FACTOR}
```

Comment:
this is the typesafe config fallback notation. if an environment variable `TEST_TIME_FACTOR` is set, this value is used, otherwise it falls back to 10.

Reply:
I see, thanks for elaborating!